library/bigip_ssl_certificate.py

```diff
@@ -0,0 +1,449 @@
+#!/usr/bin/python
+
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# TODO: pkcs12 support when f5-sdk adds it, or convert locally to pem?
+#       Would be nice if f5-sdk file upload could take a string like soap
+#       cert/key import, then we could convert in memory and import
+# TODO: Exception Handling
+
+from ansible.module_utils.basic import *
+
+DOCUMENTATION = '''
+module: bigip_ssl_certificate
+short_description: Import/Delete certificates from BIG-IP
+description:
+    - This module will import/delete SSL certificates on BIG-IP LTM
+      systems from a certificate file on the filesystem where the playbook
+      is run from.  Currently this only supports certificates to be used
+      within SSL Profiles.  PEM format with certificate and key in seperate
+      files is icurrently the only supported format.
+options:
```

Comment:
The `DOCUMENTATION` and related blocks at the tops of these files can be spaced with 2 spaces.